### PR TITLE
[EdgeTPU] Remove the onecc codes in index.js file

### PR DESCRIPTION
**Pull request by hohee-hee**
- Remove the useless code
- Modify setDefaultValues() : input-path has postfix '.tflite', output-path has postfix '_edgetpu.tflite'

ONE-vscode-DCO-1.0-Signed-off-by: hohee-hee <khappy517@gmail.com>

### CODE REVIEW

**Comment by rhqjatn2398** [comment]
There is a typo in the PR title.

as-is:
[EdgeTPU] Remove the onecc codes in indes.js file

to-be:
[EdgeTPU] Remove the onecc codes in index.js file

**Comment by hohee-hee** [comment]
> There is a typo in the PR title.
> 
> as-is: [EdgeTPU] Remove the onecc codes in indes.js file
> 
> to-be: [EdgeTPU] Remove the onecc codes in index.js file

OMG I modifed PR title but should I also modify my commit msg's title?

**Comment by rhqjatn2398** [comment]
It's probably okay because this PR will be squashed and merged.

File: media/EdgetpuCfgEditor/index.js [hunk 1]
```diff
@@ -120,267 +66,45 @@ function main() {
...
-}
-
 function registerCompilerStep() {
   const checkboxEdgeTPUCompile = document.getElementById("checkboxEdgeTPUCompile");
   const checkboxEdgeTPUProfile = document.getElementById("checkboxEdgeTPUProfile");
   const stepEdgeTPUCompile = document.getElementById("stepEdgeTPUCompile");
   const stepEdgeTPUProfile = document.getElementById("stepEdgeTPUProfile");
 
   checkboxEdgeTPUCompile.addEventListener("click", function () {
+    console.log("compile event clicked");
```
Comment (rhqjatn2398) [inline media/EdgetpuCfgEditor/index.js]: I think we should remove this logging code in this PR or later.

File: media/EdgetpuCfgEditor/index.js [hunk 2]
```diff
@@ -120,267 +66,45 @@ function main() {
...
     updateEdgeTPUStep();    
     applyUpdates();
   });
   checkboxEdgeTPUProfile.addEventListener("click", function () {
+    console.log("profile event clicked");
```
Comment (rhqjatn2398) [inline media/EdgetpuCfgEditor/index.js]: I think we should remove this logging code in this PR or later.

File: media/EdgetpuCfgEditor/index.js [hunk 3]
```diff
@@ -120,267 +66,45 @@ function main() {
...
     updateEdgeTPUStep();    
     applyUpdates();
   });
- 
+  
   stepEdgeTPUCompile.addEventListener("click", function () {
+    console.log("compile step event clicked");
```
Comment (rhqjatn2398) [inline media/EdgetpuCfgEditor/index.js]: I think we should remove this logging code in this PR or later.

File: media/EdgetpuCfgEditor/index.js [hunk 4]
```diff
@@ -120,267 +66,45 @@ function main() {
...
     updateStepUI("EdgeTPUCompile");
   });
   stepEdgeTPUProfile.addEventListener("click", function () {
+    console.log("profile step event clicked");
```
Comment (rhqjatn2398) [inline media/EdgetpuCfgEditor/index.js]: I think we should remove this logging code in this PR or later.

Comment (hohee-hee) [reply media/EdgetpuCfgEditor/index.js]: Thanx, I removed all and committed. Plz check it.

**Review by rhqjatn2398 — APPROVED** [review]
LGTM 👍

**Pull request merged**